core/src/test/java/com/google/googlejavaformat/java/FormatterTest.java

```diff
@@ -438,7 +438,7 @@ public void indentsIfStatementsWithoutBracesButWithCommentsOnTheSameLine()
                 + "\n"
                 + "  public boolean compare(ProglangType other) {\n"
                 + "    if (this == other) // comment\n"
-                + "     return true;\n"
+                + "        return true;\n"
```

Comment:
Hmm, IIRC, the convention is that most lines of code are indented +2 rather than +4 under google-java-format and the [Google Java Style guide](https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation). Have I missed something here?